.github/workflows/ci.yaml

```
@@ -11,11 +11,12 @@ on:
 jobs:
   test:
     name: Test
-    runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.10", "3.11"]
+        os: [ windows-latest, ubuntu-22.04 ]
```

Please avoid `latest` and specify the version. 
```suggestion
        os: [ windows-2022, ubuntu-22.04 ]
```